qadence_protocols/types.py

```diff
@@ -25,3 +25,9 @@ class Protocols(StrEnum):
 
 
 qadence_available_protocols = Protocols.list()
+
+
+class ReadOutOptimization(StrEnum):
+    MLE = "mle"
+    CONSTRAINED = "constrained"
+    MTHREE = "mthree"  ## matrix free measurement mitigation
```

Comment:
```suggestion
    MTHREE = "mthree" 
    # Matrix free measurement mitigation
```

Reply:
Moved the comment above and left a line spacing between the variable declarations